.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,62 @@
+name: Bug Report
+description: A clear and concise description of what the bug is.
+title: "[Bug]: "
+labels: ["bug"]
+
+body:
+  - type: markdown
+  attributes:
+  value: |
+    Thank you for helping us improving SORA!
+    
+  - type: textarea
+  id: preconditions
+  attributes:
+    label: Preconditions
+    description: What are the preconditions to reproduce the bug ?
+  validations:
+    required: true
```

Comment:
I believe that precondition shouldn't be the mandatory field. We can have bugs without preconditions, for example "swap returns error to all users"